client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dataviews/index.tsx

```diff
@@ -58,20 +123,26 @@ const SitesDataViews = ( { data, isLoading, onViewChange }: SitesDataViewsProps
 				);
 			}
 		},
-		[ sites, isLoading ]
+		[ isLoading ]
 	);
 
 	const fields = [
 		{
 			id: 'status',
-			header: '',
+			header: translate( 'STATUS' ),
```

Comment:
`Status`

<img width="311" alt="image" src="https://github.com/Automattic/wp-calypso/assets/9832440/2a1a2ecd-a270-4796-af8c-20f02f28ef41">

A sample from the DataViews (Gutenberg experiment): 
![image](https://github.com/Automattic/wp-calypso/assets/9832440/0dcdc7cd-6490-4ae1-a2ee-87e2f62ccc2b)

Reply:
I'd remove the condition selector if DataViews allows it. So, by default, the condition will always be `is`; there won't be more options for it.

Reply:
> I'd remove the condition selector if DataViews allows it.

Yes that's one of the things that was still to do. Added now (plus the title case change). Thanks for reminding me about the Gutenberg demo, that helped me find the solution quickly.

> But I think it's not customizable to set the filter value in bold. But I think we can remove the condition (is)

I might be misunderstanding this. Are you referring to the option to select is / is not? That I've just removed, but I'm also wondering if you're referring to the word 'is'? For example 'Status Backup Failed'?

I've capitalized all the words in each status type so that status reads better for now though (eg. 'Status is Backup Failed').

Commit: b6792bb0cfcbb83ca7ab8f3f27672aa1f4e23e39

Reply:
![Uploading image.png…]()